docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -198,22 +198,49 @@ streams:
           saturation_threshold: 0.95
   ...
 
-It configures the overload manager to reset certain streams depending on the
-heap size.  When the heap usage is less than 85%, no streams will be reset.
-When heap usage is at or above 85%, we start to reset certain memory classes
-(e.g. streams using memory within a power of two range). There are 8 buckets,
-with the last bucket capturing all of the streams using :math:`>= 128 *
-minimum_threshold_for_tracking`.  In this example the `minimum_threshold_for_tracking` is 1MB.
+By setting the `minimum_account_to_track_power_of_two` to `20`, we will only track
+streams using >= 1MB worth of allocated memory in buffers. Streams using >= 1MB
+will be classified into 8 power of two sized buckets. For this example, the
```

Comment:
Is it 8 regardless of the value of `minimum_account_to_track_power_of_two `? Not clear from the docs

Reply:
Done